### DISCUSSION TITLE
ssl-mode

### DISCUSSION THREAD

**Discussion by twskopos** [Sequel-Ace/Sequel-Ace#1347] [Q&A]
Hi,
I wonder if it is possible to add -ssl-mode=REQUIRED to the connection specification as it is possible with MySQLWorkbench.
It is needed with one of my connections.
Any hints are appreciated
Thanks, Tom

**Comment by Jason-Morcos** [comment]
Out of curiosity, does checking "connect using SSL" in the connection's setup not cause everything to work properly?
